google-http-client-test/src/main/java/com/google/api/client/test/json/AbstractJsonFactoryTest.java

```diff
@@ -335,22 +372,27 @@ public static class NumberTypesAsString {
     @Key @JsonString Map<String, Long> longMapValue;
   }
 
-  static final String NUMBER_TYPES =
-      "{\"bigDecimalValue\":1.0,\"bigIntegerValue\":1,\"byteObjValue\":1,\"byteValue\":1,"
-          + "\"doubleObjValue\":1.0,\"doubleValue\":1.0,\"floatObjValue\":1.0,\"floatValue\":1.0,"
-          + "\"intObjValue\":1,\"intValue\":1,\"longListValue\":[1],\"longMapValue\":{\"a\":1},"
-          + "\"longObjValue\":1,\"longValue\":1,\"shortObjValue\":1,\"shortValue\":1,"
-          + "\"yetAnotherBigDecimalValue\":1}";
+  static final String NUMBER_TYPES;
+  static final String NUMBER_TYPES_AS_STRING;
 
-  static final String NUMBER_TYPES_AS_STRING =
-      "{\"bigDecimalValue\":\"1.0\",\"bigIntegerValue\":\"1\",\"byteObjValue\":\"1\","
-          + "\"byteValue\":\"1\",\"doubleObjValue\":\"1.0\",\"doubleValue\":\"1.0\","
-          + "\"floatObjValue\":\"1.0\",\"floatValue\":\"1.0\",\"intObjValue\":\"1\","
-          + "\"intValue\":\"1\",\"longListValue\":[\"1\"],\"longMapValue\":{\"a\":\"1\"},"
-          + "\"longObjValue\":\"1\",\"longValue\":\"1\","
-          + "\"shortObjValue\":\"1\","
-          + "\"shortValue\":\"1\",\"yetAnotherBigDecimalValue\":\"1\"}";
+  static {
```

Comment:
I can see how this may be needed for more complex setup or when initializing a class (which can sometimes result in `NoClassDefFoundError`)  but do we still need this static block if we're only initializing a string constant?

Reply:
Indeed not, I think it was a wrong conclusion from solving a NoClassDefFoundError while preparing this PR that I extrapolated to the whole repo. I reverted this so only actual complex logic is moved into a static block with trycatch (only 2 cases)

Reply:
Thank you.